Packs/MalwareInvestigationAndResponse/Scripts/KillProcessWrapper/KillProcessWrapper.yml

```diff
@@ -0,0 +1,46 @@
+args:
+- default: false
+  description: ' The Endpoint ID in which you would like to kill the given process.'
+  isArray: false
+  name: endpoint_id
+  required: true
+  secret: false
+- default: false
+  description: 'The ID of the process to kill. '
+  isArray: false
+  name: process_id
+  required: false
+  secret: false
+- default: false
+  description: 'The name of the process to kill. '
+  isArray: false
+  name: process_name
+  required: false
+  secret: false
+- auto: PREDEFINED
+  default: false
+  description: 'Do you confirm killing the process? '
+  isArray: false
+  name: approve_action
+  predefined:
+  - 'YES'
+  - 'NO'
+  required: true
+  secret: false
+comment: 'A cross-vendor wrapper script that triggers a ‘process kill’ command - i.e
+  executes the proper kill process command according to the vendor: CrowdstrikeFalcon
+  or Cortex XDR.'
+commonfields:
+  id: KillProcessWrapper
+  version: -1
+enabled: false
+name: KillProcessWrapper
+script: '-'
+subtype: python3
+system: false
+timeout: '0'
+type: python
+dockerimage: demisto/python3:3.10.4.29342
+fromversion: 6.2.0
```

Comment:
This cant be 5.0.0?

Reply:
As far as I know, new content should be created for versions 6.2.0 and up. 
Let me know if you know something else.